typings/rendition.d.ts

```diff
@@ -125,6 +125,15 @@ declare module 'rendition' {
 
 	class Alert extends RenderableElementWithProps<AlertProps, any> {}
 
+	interface ArcSliderProps extends BoxProps {
+		onValueChange?: (value: number) => any;
```

Comment:
This should better be `(value: number) => void` just line in the documentation.

Reply:
good call